cli/kubectl_kadalu/storage_add.py

```diff
@@ -38,17 +38,40 @@ def storage_add_args(subparsers):
         default=[],
         action="append"
     )
+    parser_add_storage.add_argument(
+        "--external",
+        help="Storage from external gluster, Example: --external gluster-node:/gluster-volname",
+        default=[],
+        action="append"
+    )
 
 
 def storage_add_validation(args):
-    num_storages = len(args.device) or len(args.path) or len(args.pvc)
+    for vol in args.external:
+        if args.type and args.type != "External":
+            print("'--external' option is used only with '--type External'",
+                  file=sys.stderr)
+            sys.exit(1)
+
+        if ":" not in vol:
+            print("Invalid external storage details. Please specify "
+                  "details in the format <node>:/<volname>", file=sys.stderr)
+            sys.exit(1)
+        # Set type to External as '--external' option is provided
+        args.type = "External"
+
+    if not args.type:
+        args.type = "Replica1"
+
+    num_storages = len(args.device) or len(args.path) or len(args.pvc) or len(args.external)
     if num_storages == 0:
         print("Please specify atleast one storage", file=sys.stderr)
         sys.exit(1)
 
     if (args.type == "Replica1" and num_storages != 1) or (
-            args.type == "Replica3" and num_storages != 3
-    ):
+            args.type == "Replica3" and num_storages != 3) or (
+                args.type == "External" and num_storages != 1
```

Comment:
If multiple allowed then change the comparison to `num_storages >= 1`

Reply:
nope, not multiple.